src/chat/chat_panel.rs

```diff
@@ -19,20 +19,21 @@ use super::{
 };
 
 live_design! {
-    import makepad_widgets::base::*;
-    import makepad_widgets::theme_desktop_dark::*;
-
-    import crate::shared::styles::*;
-    import crate::shared::widgets::*;
-    import makepad_draw::shader::std::*;
-
-    import crate::chat::model_selector::ModelSelector;
-    import crate::chat::chat_line::ChatLine;
-    import crate::chat::shared::ChatModelAvatar;
-    import crate::chat::shared::ChatAgentAvatar;
-    import crate::chat::prompt_input::PromptInput;
+    use link::theme::*;
+    use link::shaders::*;
+    use link::widgets::*;
+
+    use crate::shared::styles::*;
+    use crate::shared::widgets::*;
+    use crate::chat::model_selector::ModelSelector;
+    use crate::chat::chat_line::ChatLine;
+    use crate::chat::shared::ChatAgentAvatar;
+    use crate::chat::shared::ChatModelAvatar;
+    use crate::chat::prompt_input::PromptInput;
 
     ICON_JUMP_TO_BOTTOM = dep("crate://self/resources/icons/jump_to_bottom.svg")
+    ICON_PROMPT = dep("crate://self/resources/icons/prompt.svg")
+    ICON_STOP = dep("crate://self/resources/icons/stop.svg")
```

Comment:
These lines are unused here and are not pub. Just a leftover of conflict resolution or I'm missing something?